From 466baa842cadef11a6995caabbf8ad1294f87426 Mon Sep 17 00:00:00 2001
From: samaga Prasanna Krishna <samaga.krishna@intel.com>
Date: Wed, 3 Apr 2013 17:59:20 +0530
Subject: [PORT FROM MAIN]Reverting the patch for bz 93519

BZ: 97715

Reverting the Temp Fix provided for bug 93519.
Acquiring the wakelock in offload HAL to fix this issue.

Change-Id: Ibacdfd8489e8ec0646cd04c6a012db56468f6c4b
Orig-Change-Id: Iacaefdbdd515740fca2aef63bf810c0778e46af7
Signed-off-by: samaga Prasanna Krishna <samaga.krishna@intel.com>
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
---
 media/libstagefright/AwesomePlayer.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/media/libstagefright/AwesomePlayer.cpp b/media/libstagefright/AwesomePlayer.cpp
index 46810da..0c6f1c2 100644
--- a/media/libstagefright/AwesomePlayer.cpp
+++ b/media/libstagefright/AwesomePlayer.cpp
@@ -1530,7 +1530,7 @@ status_t AwesomePlayer::getPosition(int64_t *positionUs) {
             && (mAudioPlayer == NULL || !(mFlags & VIDEO_AT_EOS))) {
         Mutex::Autolock autoLock(mMiscStateLock);
         *positionUs = mVideoTimeUs;
-    } else if (mAudioPlayer != NULL && !(mFlags & AT_EOS)) {
+    } else if (mAudioPlayer != NULL) {
         *positionUs = mAudioPlayer->getMediaTimeUs();
     } else {
         *positionUs = 0;
-- 
1.7.4.1

